spec/integ/admin-rooms.spec.js

```diff
@@ -593,6 +647,53 @@ describe("Admin rooms", function() {
         expect(sentAckNotice).toBe(true, "sent ACK m.notice");
     }));
 
+    it("should not try to change to a nickname that is already in use", async () => {
+        const newNick = "Blurple";
+
+        // make sure that the NICK command not is sent
+        let sentNickCommand = false;
+        env.ircMock._whenClient(roomMapping.server, userIdNick, "send",
+        function() {
```

Comment:
Maybe check if the second parameter `command` equals `NICK`?
What other commands are send via `send`?

Reply:
Good catch. I believe only nick